_commands/basics/who.md

```diff
@@ -5,20 +5,66 @@ who
 --
 TODO: Add documentation for this command by submitting a pull request.
 <!-- one line explanation would go here -->
+## Description
+Displays list of all users currently logged on 
 
 <!-- minimal example -->
+### Example Output 
 ~~~ bash
-*short* example of command usage and output
+$ Username console  Oct 12 08:20 
+$ Username ttys000  Oct 20 09:35 
 ~~~
 
+### Example Command
 <!--more-->
+~~~ bash
+$ who -b 
+reboot ~ Oct 12 8:20
+
+$ who -T
+Username - console  Oct 12 08:20 
+Username + ttys000  Oct 20 20:04 
 
-### Useful Options / Examples
+$ who -u
+Username console  Oct 12 08:20  old  	    99
+Username ttys000  Oct 20 20:04   .   	 87780
+
+~~~
 
-#### Example command
+ The who utility displays a list of all users currently logged on, showing for each user the login name, tty
+ name, the date and time of login, and hostname if not local.
 
-##### Break it down
+#### Available options:
```

Comment:
Instead of copy/pasting the man page here, could you pick a few of the options and show an example of them and describe why they might be useful? That will be a bit more valuable since everyone has access to the `man` pages. It's not as easy finding out which options are most used/useful and why they are used so providing some of that information here is super helpful.